common/presenters/populationTable/location-meta-to-spaces-link.js

```diff
@@ -0,0 +1,19 @@
+const spacesFormatter = function (count) {
+  if (count === undefined) {
+    return 'Add'
+  } else if (count === 0) {
+    return '0 spaces'
```

Comment:
This also feels like a scenario where you can use the [i18next library's plural functionality](https://www.i18next.com/translation-function/plurals) that we've used elsewhere.

That would also keep the consistency of keeping content to the translations.